chalicelib/checks/wrangler_checks.py

```diff
@@ -560,7 +560,7 @@ def change_in_item_counts(connection, **kwargs):
     total_counts_db = sum([diff_counts[coll]['DB'] for coll in diff_counts if diff_counts[coll]['DB'] >= 0])
     # see if we have negative counts
     # allow negative counts, but make note of, for the following types
-    purged_types = ['tracking_item', 'higlass_view_config']
+    purged_types = ['TrackingItem', 'HiglassViewConfig']
```

Comment:
This strongly argues for taking the names of the tags and separating them out into constants. Might be better to do that in an organized way, so perhaps a task for another day. But I just wanted to say it here to have said it. I'd settle for a `# TODO` comment perhaps?

Reply:
Agreed, I'll add a comment.